src/damavand/cloud/aws/controllers/spark.py

```diff
@@ -24,11 +24,13 @@ def __init__(
         **kwargs,
     ) -> None:
         super().__init__(name, applications, tags, **kwargs)
+        print("\n\n\nShit")
```

Comment:
This has to be removed

Reply:
I don't debug like this; I think it was left from my Copilot :))